core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -10,6 +10,20 @@
  */
 public class MerlinsBeard {
 
+    public enum NetworkType {
```

Comment:
we're in library world here, don't use enums and take advantage of the typed annotations that Android provides.

:exclamation: https://medium.com/google-developers/developing-for-android-i-understanding-the-mobile-context-fd2351b131f8